README.md

```diff
@@ -222,16 +222,34 @@ npx create-react-app test-template --template file:./carto-react-template/templa
 
 ### Publishing the template
 
-REMEMBER TO COPY the `hygen/_templates` folder.
+> ⚠️ Important: remember to set the right version for each template, tag the release in the GitHub repository and deploy the demo web app with the latest changes
 
-> ⚠️ Important: remember to synchronize the changes applied to your `template/package.json` with `template/package.dev.json` and `template.json` and remove the `template/package.json` file before publishing.
+For each template, execute the release command from their **base folder**. This folder will clean all unnecesary development files and folders and copy the hygen templates.
 
-To publish this template execute npm publish from the **root directory** of this project.
+```bash
+cd template-sample-app
+yarn release
+```
+
+
+### Deploying the demo web
+
+The demo web app is hosted in Firebase, so before deploying it you'll need to log into Firebase using:
```

Comment:
In addition to the sample app, aren't we also deploying the storybook using Firebase?

Reply:
Yes, but this is made from the react-lib repository

Reply:
OK